### ISSUE TITLE
qodana.yaml file isn't read and load

### ISSUE THREAD

**Issue by LocalHeroPro** [JetBrains/Qodana#14]
Steps to reproduce:
1. Create `qodana.yaml` file in the same folder where you point docer - for me it's `-v /var/www/site.local/html/`:
```yaml
version: 1.0
failThreshold: 0
profile:
  name: qodana.site
exclude:
  - name: All
    paths:
      - public
      - storage
      - vendor

```
2. Run docker:
```bash
docker run --rm -it -v /var/www/site.local/html/:/data/project/ -p 8080:8080 jetbrains/qodana --show-report
```
I get 
```bash
// ...
Starting up IntelliJ IDEA 2020.3.2 (build IJCA-203.6906) ...done.
Opening project...done.
Initializing project...Loaded profile 'qodana.recommended' from file '/data/profile.xml'
Inspecting with profile 'qodana.recommended'
Analyzing code 0%
// ...
```
So I can't see that my custom `qodana.yaml` file is imported ;/
Also, can I check in the browser what settings are imported? Or or what name of profile is imported?

**Comment by a-kuznetsova** [comment]
Hi @LocalHeroPro,

Are you still experiencing this issue with the `jetbrains/qodana:2021.1-eap` version?

Thank you!

**Comment by LocalHeroPro** [comment]
```php
Listening for transport dt_socket at address: 5005
WARNING: An illegal reflective access operation has occurred
WARNING: Illegal reflective access by com.intellij.util.ui.StartupUiUtil to constructor com.sun.java.swing.plaf.gtk.GTKLookAndFeel()
WARNING: Please consider reporting this to the maintainers of com.intellij.util.ui.StartupUiUtil
WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
WARNING: All illegal access operations will be denied in a future release
[    814][ApplicationImpl pooled thread 8]   WARN - j.internal.DebugAttachDetector - Unable to start DebugAttachDetector, please add `--add-exports java.base/jdk.internal.vm=ALL-UNNAMED` to VM options 
Starting up IntelliJ IDEA 2021.1 Beta (build IJCA-211.6432.7) ...done.
Opening project...Python environment configuration...
Python interpreters detected:
/usr/bin/python3.8
/usr/bin/python3
Settings up interpreter /usr/bin/python3.8
Waiting on event queue... 57%

```

**Comment by a-kuznetsova** [comment]
Is there a `qodana.site` profile defined in your project? Look at [this](https://github.com/JetBrains/ideavim/blob/master/.idea/inspectionProfiles/Qodana.xml)  example of the `Qodana` profile.

Otherwise, if you want to use a default qodana profile, use `qodana.recommended`.

**Comment by a-kuznetsova** [comment]
Hi @LocalHeroPro,

I am closing this issue because it has been inactive for almost two weeks.

Please open new issue via [our issue tracker](https://youtrack.jetbrains.com/newIssue?project=QD) if you still encounter this problem.


Thank you!
